import-export-cli/integration/testutils/throttlePolicy_testUtils.go

```diff
@@ -0,0 +1,309 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package testutils
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/apim"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/base"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+const (
+	policyIDKey   = "policyId"
+	policyNameKey = "policyName"
+	policyIDReset = "-1"
+
+	applicationPolicySubType  = "application policy"
+	advancedPolicySubType     = "advanced policy"
+	customPolicySubType       = "custom rule"
+	subscriptionPolicySubType = "subscription policy"
+)
+
+// ValidateThrottlePolicyExportImport : Validates Exporting Throttling Policy from source env and Importing to destination env
+func ValidateThrottlePolicyExportImport(t *testing.T, args *ThrottlePolicyImportExportTestArgs, policyType string) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[policyNameKey])
+
+	exportedOutput, err := exportThrottlePolicy(t, policyName, args.SrcAPIM.GetEnvName())
+	assert.Nil(t, err, "Error while exporting the Throttling Policy")
+
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+	args.SrcAPIM.DeleteThrottlePolicy(fmt.Sprintf("%v", args.Policy[policyIDKey]), policyType)
+
+	// Import Throttling Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	importedOutput, err := importThrottlePolicy(t, args)
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully imported")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, policyName, policyType)
```

Comment:
Don't we have to use `args.DesAPIM.GetThrottlingPolicyByName` since it something binds with the environment? We can implement this function in `apim.go` right? FYI @wasuradananjith